vector/src/main/java/im/vector/app/features/location/LocationSharingService.kt

```diff
@@ -140,6 +145,25 @@ class LocationSharingService : VectorService(), LocationTracker.Callback {
         }
     }
 
+    private fun updateStoppedBeaconInfo(roomId: String) {
+        activeSessionHolder
+                .getSafeActiveSession()
+                ?.let { session ->
+                    session.coroutineScope.launch(session.coroutineDispatchers.io) {
+                        val room = session.getRoom(roomId)
+                        EventType
```

Comment:
I think we also need to add a filter to get all state events with `lived=true`. Normally there should be only a single live beacon per user per room. Maybe we need to extract it into a dedicated method to use it also in `sendLiveLocation`. I see in this last method 2 missing filters: the state_key=userId and lived=true which will be solved by this new method.

Reply:
Moved to an helper function.